istioctl/cmd/remove-from-mesh.go

```diff
@@ -187,6 +187,10 @@ func unInjectSideCarFromDeployment(client kubernetes.Interface, deps []appsv1.De
 			dep.Name, dep.Namespace)
 		res := dep.DeepCopy()
 		depName := strings.Join([]string{dep.Name, dep.Namespace}, ".")
+		if dep.Name == istioEgressgateway || dep.Name == istioIngressgateway {
+			fmt.Fprintf(writer, "failed to remove the deployment %q from the mesh. remove-from-mesh just removes the sidecar container. Skipping.\n", depName)
```

Comment:
Can also `multierror.Append` here.

Reply:
I think so. Thanks for your comments. Let us consider  @howardjohn  point of view.

Reply:
see https://github.com/istio/istio/issues/25860#issuecomment-664089111